src/main/scala/com/atomist/rug/kind/core/ProjectMutableView.scala

```diff
@@ -99,9 +100,11 @@ class ProjectMutableView(
   @ExportFunction(readOnly = true,
     exposeAsProperty = true,
     description = "The total number of files in this project")
+  @Deprecated
```

Comment:
Why is this deprecated? I think this is arguably a primitive because it may be more efficient